proposals/proposal-template.md

```diff
@@ -0,0 +1,19 @@
+> Note to proposers: Please keep this document as brief as possible, preferably not more than two pages.
```

Comment:
Hi, some comments about this template:

- rename the file name to `TEMPLATE.md`, as it is in `proposals` directory.
- add some Headers about the status:
```markdown
Status: Pending | Draft | Completed
Version: Alpha | Beta | GA
Implementation Owner: TBD
```

Reply:
I am a bit unclear on `Status` - if the proposal is merged, then presumably it's complete, otherwise it is under review.

Also, `Owner` - My take would be that the community at large "owns" any accepted proposal